site/content/en/docs/components/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Components"
+weight: 30
+---
+
+## Prow Images
+
+This directory includes a sub directory for every Prow component and is where all binary and container images are built. You can find the `main` packages here. For details about building the binaries and images see ["Building, Testing, and Updating Prow"](/docs/build-test-update/).
+
+## Cluster Components
+
+Prow has a microservice architecture implemented as a collection of container images that run as Kubernetes deployments. A brief description of each service component is provided here.
+
+#### Core Components
+
+* [`crier`](/docs/components/core/crier/) reports on ProwJob status changes. Can be configured to report to gerrit, github, pubsub, slack, etc.
```

Comment:
I have a question: will the Prow source codes be migrated too,
from https://github.com/kubernetes/test-infra to https://github.com/kubernetes-sigs/prow (or somewhere else)?

Related issue:

[Before]
This `crier` link was pointing to https://github.com/kubernetes/test-infra/tree/master/prow/cmd/crier ,
and when a user follows that link, then he/she can
1. read the `crier` doc, because the GitHub UI displays the `README.md`
2. check the `crier` source code

in that page.

[After]
This `crier` link points to https://deploy-preview-26--k8s-prow.netlify.app/docs/components/core/crier/ ,
where a user can find `crier` doc, but not `crier` source code.

One alternative option might be:
```suggestion
* `crier` ([doc](/docs/components/core/crier/), [code](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/crier)) reports on ProwJob status changes. Can be configured to report to gerrit, github, pubsub, slack, etc.
```

Reply:
> I have a question: will the Prow source codes be migrated too,
from https://github.com/kubernetes/test-infra to https://github.com/kubernetes-sigs/prow (or somewhere else)?

You just asked the million-dollar question. Yes we want to do this (everyone agrees this should be done, last I checked), but currently the effort is not staffed.

The suggested change is the right approach (i.e., link to kubernetes/test-infra from this repo for any code links).